frontend_app_kanban/src/component/KanbanCard.jsx

```diff
@@ -39,12 +39,41 @@ function KanbanCard (props) {
     )
   }
 
+  const luminance = (color) => {
```

Comment:
A good practice is to cite the source of that algorithm

can you give the url you copied it from?

Reply:
done, I link the formula